src/overlays/actors/ovl_En_Kaizoku/z_en_kaizoku.c

```diff
@@ -93,63 +104,75 @@ Vec3f D_80B8AA04[] = {
     { 470.0f, 30.0f, 140.0f },   { 410.0f, 80.0f, 130.0f },   { 410.0f, 80.0f, 130.0f },   { 0.0f, 0.0f, 0.0f },
 };
 
-Color_RGBA8 D_80B8AB24[] = {
-    { 255, 255, 90, 255 },
-    { 55, 25, 80, 255 },
-    { 5, 195, 110, 255 },
+typedef enum KaizokuLipstickColors {
+    /* 0x0 */ KAIZOKU_LIPSTICK_YELLOW,
+    /* 0x1 */ KAIZOKU_LIPSTICK_PURPLE,
+    /* 0x2 */ KAIZOKU_LIPSTICK_SEAGREEN, // (between aqua and green)
+} KaizokuLipstickColors;
+
+Color_RGBA8 sKaizokuLipstickColors[] = {
+    { 255, 255, 90, 255 }, // KAIZOKU_LIPSTICK_YELLOW
+    { 55, 25, 80, 255 },   // KAIZOKU_LIPSTICK_PURPLE
+    { 5, 195, 110, 255 },  // KAIZOKU_LIPSTICK_SEAGREEN
 };
 
-Color_RGBA8 D_80B8AB30[] = {
-    { 255, 130, 10, 255 },
-    { 185, 130, 210, 255 },
-    { 135, 195, 80, 255 },
+typedef enum KaizokuOutfitColors {
+    /* 0x0 */ KAIZOKU_OUTFIT_ORANGE,
+    /* 0x1 */ KAIZOKU_OUTFIT_LAVENDER,
+    /* 0x2 */ KAIZOKU_OUTFIT_GREEN,
+} KaizokuOutfitColors;
+
+static Color_RGBA8 sKaizokuOutfitColors[] = {
+    { 255, 130, 10, 255 },  // KAIZOKU_OUTFIT_ORANGE
+    { 185, 130, 210, 255 }, // KAIZOKU_OUTFIT_LAVENDER
+    { 135, 195, 80, 255 },  // KAIZOKU_OUTFIT_GREEN
 };
 
-typedef enum EnKaizokuDamageEffect {
+typedef enum KaizokuDamageEffect {
     /* 0x0 */ KAIZOKU_DMGEFF_NONE,
-    /* 0x1 */ KAIZOKU_DMGEFF_1,
-    /* 0x2 */ KAIZOKU_DMGEFF_2,
+    /* 0x1 */ KAIZOKU_DMGEFF_STUNNED,
+    /* 0x2 */ KAIZOKU_DMGEFF_FIRE_ARROW,
     /* 0x3 */ KAIZOKU_DMGEFF_FREEZE,
-    /* 0x4 */ KAIZOKU_DMGEFF_4,
-    /* 0x5 */ KAIZOKU_DMGEFF_5,
-    /* 0xD */ KAIZOKU_DMGEFF_D = 0xD,
-    /* 0xE */ KAIZOKU_DMGEFF_E,
-    /* 0xF */ KAIZOKU_DMGEFF_F
-} EnKaizokuDamageEffect;
+    /* 0x4 */ KAIZOKU_DMGEFF_LIGHT_ARROW,
+    /* 0x5 */ KAIZOKU_DMGEFF_ZORA_SHIELD,
+    /* 0xD */ KAIZOKU_DMGEFF_STUNNED_ONLY = 0xD, // smashed? it checks stun first?
```

Comment:
```suggestion
    /* 0xD */ KAIZOKU_DMGEFF_STUNNED_ONLY = 0xD, // deals damage only if the pirate is stunned; she jumps away otherwise
```
I feel like this comment could be a little better at describing what this effect does. I'm not picky about the wording, so feel free to reword it if you'd like; anything that describes this piece of logic in one sentence will do:
```c
            case KAIZOKU_DMGEFF_STUNNED_ONLY:
                if (this->action == KAIZOKU_ACTION_STUNNED) {
                    wasHit = true;
                } else if (this->action != KAIZOKU_ACTION_JUMP) {
                    EnKaizoku_SetupJump(this);
                }
                break;
```